### PR TITLE
Add code coverage with github actions

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

**Comment by jermnelson** [comment]
I think we want to exclude the `tests` directory and the `folio_migration_tools` repository. I created a  `.coveragerc` file with with contents to  excludes these directories:
```
[run]
omit = 
    plugins/tests/*
    *migration_tools/*
```

**Review by jermnelson — CHANGES_REQUESTED** [review]
Looks good. Just a couple of observations about the code coverage picking up the `folio_migration_tools` code and I think we want to exclude the tests. Also do we need the sqlite database `.coverage` file checked in the source code?

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -33,4 +34,6 @@ jobs:
         git clone https://github.com/FOLIO-FSE/folio_migration_tools.git /Users/Shared/folio_migration_tools --depth=2
```
[the remark this answers is not in the document]
Comment (jgreben) [reply .github/workflows/python-app.yml:34]: Ok so I added the .coveragerc file. That should remove `folio_migration_tools` from the coverage, right? I need to keep the `Install support repos` task in the workflow for the test to be able to run. Coveralls is not updating right now, so we can't see the most recent build.

[the remark this answers is not in the document]
Comment (jgreben) [reply .github/workflows/python-app.yml:34]: Also I removed the `.coverage` file.

**Review by jermnelson — APPROVED** [review]

**Pull request merged**